src/views/dissolved-search/index.html

```diff
@@ -2,10 +2,17 @@
 
 {% block content %}
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-three-quarters">
     <h1 class="govuk-heading-xl">
       Dissolved company search
```

Comment:
Should the heading be `Search for a dissolved company` like in the prototype?

Reply:
Good point, amended